src/main/resources/sessionLinksRecoveryEmailTemplate-feedbackSessionAccessLinks.html

```diff
@@ -0,0 +1,26 @@
+<p>Hello ${userName},</p>
+
+<p>
+    You are receiving this email because a request to resend TEAMMATES links related to this email address <strong>${recoveryEmail}</strong> was made on <a href="${teammateHomePageLink}">TEAMMATES</a>.
+</p>
+
+<p>
+    Below are the sessions that are associated with the email <strong>${recoveryEmail}</strong> within the past 180 days:
```

Comment:
>Yes, we can mention that the sessions shown are limited to those opening/closing in the past 180 days.

Pls take prof's comment into account. I think this shld be specified.

Reply:
Yes, but my written English is very bad.. Would you mind help me rephrase the sentence?